src/HttpResponse.h

```diff
@@ -75,10 +75,12 @@ struct HttpResponse : public AsyncSocket<SSL> {
         httpResponseData->state &= ~HttpResponseData<SSL>::HTTP_RESPONSE_PENDING;
     }
 
+#ifndef UWS_HTTPRESPONSE_NO_WRITEMARK
     /* Called only once per request */
     void writeMark() {
         writeHeader("uWebSockets", "v0.17");
```

Comment:
You can actually just move the macro around this one line. No need to put the macro at multiple places.

Reply:
Done.